lib/matplotlib/testing/jpl_units/UnitDblFormatter.py

```diff
@@ -6,24 +6,26 @@
 
 
 class UnitDblFormatter(ticker.ScalarFormatter):
-    """The formatter for UnitDbl data types.  This allows for formatting
-        with the unit string.
+    """
+    The formatter for UnitDbl data types.
+
+    This allows for formatting with the unit string.
     """
     def __init__(self, *args, **kwargs):
-        'The arguments are identical to matplotlib.ticker.ScalarFormatter.'
+        """The arguments are identical to matplotlib.ticker.ScalarFormatter."""
```

Comment:
this docstring and the following ones could just be inherited (in fact the whole `__init__` can be deleted and inherited...).

Reply:
Thanks! Removed all.